### PR TITLE
Build with dotnet

### PR DESCRIPTION
**Pull request by victorhurdugaci**
cc @Tratcher @ryanbrandenburg

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 init:
   - git config --global core.autocrlf true
 build_script:
-  - build.cmd --quiet --parallel verify
```
Comment (ryanbrandenburg) [inline appveyor.yml]: Is it that we don't want it parallel anymore or does that idea just not exist in the new world?

Comment (pranavkm) [inline appveyor.yml]: @ryanbrandenburg some of these come from the Universe template - https://github.com/aspnet/Universe/blob/dev/build-template-dotnet/appveyor.yml. If we find issues with these later, we'll bulk update all the repos. For now, I'd focus on code changes

File: test/Microsoft.AspNet.Antiforgery.FunctionalTests/project.json [hunk 2]
```diff
@@ -2,23 +2,24 @@
...
   "commands": {
-    "run": "xunit.runner.aspnet",
     "test": "xunit.runner.aspnet"
   },
   "frameworks": {
     "dnx451": {
       "dependencies": {
         "Moq": "4.2.1312.1622",
-        "System.Net.Http": "4.0.1-rc2-23621"
+        "System.Net.Http": "4.0.1-rc2-23621",
```
Comment (ryanbrandenburg) [inline test/Microsoft.AspNet.Antiforgery.FunctionalTests/project.json]: I know it was that way before but is this hyper-specific for a reason?

Comment (victorhurdugaci) [inline test/Microsoft.AspNet.Antiforgery.FunctionalTests/project.json]: Not sure about that...

**Pull request merged**